src/main/java/roomescape/reservation/domain/Reservation.java

```diff
@@ -0,0 +1,63 @@
+package roomescape.reservation.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDate;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import roomescape.common.exception.ReservationException;
+import roomescape.member.domain.Member;
+import roomescape.theme.domain.Theme;
+
+@Getter
+@ToString
+@NoArgsConstructor
+@Entity
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    private Member member;
+    private LocalDate date;
```

Comment:
빈 줄을 추가해 가독성을 높일 수 있겠네요!
```suggestion
    private Member member;
    
    private LocalDate date;
```

Reply:
감사합니다!